README.md

```diff
@@ -259,10 +275,21 @@ To run freeipa management service from command line you have to run the below gr
 -Dspring.config.location=$(pwd)/freeipa/src/main/resources/application.yml,$(pwd)/freeipa/build/resources/main/application.properties"
 ````
 
-## Database development
+### Environment
+To run environment service from command line you have to run the below gradle command with the following list of JVM parameters:
+
+ ````
+./gradlew :environment:bootRun -PjvmArgs="\
+-Denvironment.db.addr=localhost \
+-Dserver.port=8088 \
+-Dvault.addr=localhost \
+-Dvault.root.token=<TOKEN_FROM_PROFILE_FILE> \
+-Dspring.config.location=$(pwd)/environment/src/main/resources/application.yml,$(pwd)/environment/build/resources/main/application.properties"
+````
 
-If any schema change is required in Cloudbreak database (cbdb), then the developer needs to write SQL scripts to migrate the database accordingly. The schema migration is managed by [MYBATIS Migrations](https://github.com/mybatis/migrations) in Cloudbreak and the cbd tool provides an easy-to-use wrapper for it. The syntax for using the migration commands is `cbd migrate <database name> <command> [parameters]` e.g. `cbd migrate migrate status`.
+## Database development
 
+If any schema change is required in Cloudbreak services databases (cbdb/environmentdb/datalakedb/redbeamsdb), then the developer needs to write SQL scripts to migrate the database accordingly. The schema migration is managed by [MYBATIS Migrations](https://github.com/mybatis/migrations) in Cloudbreak and the cbd tool provides an easy-to-use wrapper for it. The syntax for using the migration commands is `cbd migrate <database name> <command> [parameters]` e.g. `cbd migrate status`.
```

Comment:
I think the example command at the end should include a specific service name, eg.: `cbd migrate cbdb status`

Plain `cbd migrate status` gives error: `Invalid database service name: status.`